gtsfm/visualization/view_scene.py

```diff
@@ -63,7 +69,73 @@ def view_scene(args: argparse.Namespace) -> None:
     for i in range(len(wTi_list)):
         wTi_list[i] = zcwTw.compose(wTi_list[i])
 
-    draw_scene_open3d(point_cloud, rgb, wTi_list, calibrations, args)
+    wTi_list_gt, gt_calibrations = _load_common_gt_poses(args, img_fnames)
+
+    if args.gt_olsson_dir is not None or args.gt_colmap_dir is not None:
+        if len(gt_calibrations) == 1:
+            gt_calibrations = gt_calibrations * len(img_fnames)
+
+        for i in range(len(wTi_list_gt)):
+            wTi_list_gt[i] = zcwTw.compose(wTi_list_gt[i])
```

Comment:
Why are we applying this transform to GT poses, but not the points?

Reply:
Because I've set it up to only visualize GT poses, not GT points. Otherwise gets a bit too crowded in addition to the reconstructed points and reconstructed poses.